src/test/java/org/ow2/proactive/connector/iaas/fixtures/CredentialsFixtures.java

```diff
@@ -25,17 +25,30 @@
  */
 package org.ow2.proactive.connector.iaas.fixtures;
 
-import org.ow2.proactive.connector.iaas.model.Credentials;
+import org.ow2.proactive.connector.iaas.model.InfrastructureCredentials;
+import org.ow2.proactive.connector.iaas.model.InstanceCredentials;
 
 
 public class CredentialsFixtures {
 
-    public static Credentials getCredentials(String username, String password) {
-        return new Credentials(username, password, "");
+    public static InstanceCredentials getInstanceCredentials(String username, String password) {
+        return new InstanceCredentials(username, password, null, null);
```

Comment:
Well, good question it seems that Intellij complains about passing Optional arguments:
```
Reports any uses of java.util.Optional<T>, java.util.OptionalDouble, java.util.OptionalInt, java.util.OptionalLong or com.google.common.base.Optional as the type for a field or a parameter. Optional was designed to provide a limited mechanism for library method return types where there needed to be a clear way to represent "no result". Using a field with type java.util.Optional is also problematic if the class needs to be Serializable, which java.util.Optional is not.
```
We also can use empty strings instead but considering the REST API (where Instance and Infrastructure objects are imported from JSON) passing null args allows to reproduce the situation where the JSON parameters are actually missing. Here to avoid NPE the arguments are wrapped with Optional.ofNullable() on the other side, although I don't know if its actually a good practice.